CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ Mapbox welcomes participation and contributions from everyone.
 #### Features
 
 #### Bug fixes and improvements
+- Added async callback for audio requests. [#5652](https://github.com/mapbox/mapbox-navigation-android/pull/5652)
+- Added `AudioFocusOwner`. [#5652](https://github.com/mapbox/mapbox-navigation-android/pull/5652)
+- Added `ttsStreamType` to `VoiceInstructionsPlayerOptions`. [#5652](https://github.com/mapbox/mapbox-navigation-android/pull/5652)
+- Fixed issue with `KEY_PARAM_STREAM`. [#5652](https://github.com/mapbox/mapbox-navigation-android/pull/5652)
```

Comment:
Could we be more specific about which APIs this impacts, with references to the public classes that were changed? For example `Fixed issue with `KEY_PARAM_STREAM` sounds a little too generic and developers might not understand what this change is about.

Reply:
Done. This is ready for another round of 👀